gems/pending/util/MiqSshUtilV2.rb

```diff
@@ -205,6 +205,12 @@ def temp_cmd_file(cmd)
     end
   end
 
+  def self.generate_ssh_keys
+    require 'sshkey'
+    keys = SSHKey.generate
```

Comment:
Why can't these two lines just live in the `create_deployment` method until they are reused somewhere else?